services/merge.ts

```diff
@@ -0,0 +1,183 @@
+import { sparqlEscapeUri } from 'mu';
+import { updateSudo } from '@lblod/mu-auth-sudo';
+
+import { CustomError } from '../utils/custom-error';
+import { Conflict } from '../types';
+
+export async function addIsConflictingFlagToPersons(
+  conflicts: Array<Conflict>,
+): Promise<void> {
+  if (conflicts.length === 0) {
+    return;
+  }
+
+  const values = conflicts.map(
+    (conflict) =>
+      `( ${sparqlEscapeUri(conflict.conflictUri)} ${sparqlEscapeUri(conflict.personUri)})`,
+  );
+  const queryString = `
+    PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+    PREFIX person: <http://www.w3.org/ns/person#>
+    PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+    PREFIX dct: <http://purl.org/dc/terms/>
+
+    DELETE {
+      GRAPH ?g {
+        ?conflict dct:modified ?modified .
+      }
+    }
+    INSERT {
+      GRAPH ?g {
+        ?conflict ext:conflictsWith ?person  .
+        ?conflict dct:modified ?modified .
```

Comment:
should be ?now right?

Reply:
not sure if we want to update modified for this as these properties shouldn't go to ldes anyway. they are just for us